bcda/auth/okta_test.go

```diff
@@ -80,8 +80,8 @@ func (s *OktaAuthPluginTestSuite) TestGenerateClientCredentials() {
 }
 
 func (s *OktaAuthPluginTestSuite) TestOktaRevokeClientCredentials() {
-	err := s.o.RevokeClientCredentials([]byte("{}"))
-	assert.Equal(s.T(), "not yet implemented", err.Error())
+	err := s.o.RevokeClientCredentials("fakeClientID")
```

Comment:
You asked for testing suggestions at this level, so I'll pass on something that was recommended for my PR: we might try mocking in a similar fashion to `bcda_worker/main_test.go` with calls similar to its ` bbc.On("GetExplanationOfBenefitData", "12000").Return(bbc.getData("ExplanationOfBenefit", "12000"))` -- basically, checking that `auth/okta.go` sends the right arguments to the right func in `auth/client/okta_client.go`.